setup.py

```diff
@@ -47,6 +52,8 @@
         "proto-plus >= 1.10.1",
         "google-cloud-storage >= 1.26.0, < 2.0.0dev",
         "google-cloud-bigquery >= 1.15.0, < 3.0.0dev",
+        "tensorflow-cpu",
```

Comment:
This is supposed to be an optional dependency. Where should I put it?

Reply:
This should be added to extra_require below like so:

```python
extras_require={
  "full": ["pandas>=1.0.0", "tensorflow-cpu>=2.0.0"],
  "tensorboard":["tensorflow-cpu >= 2.0.0"] 
}
```

Also pin the dependency to a version.

Reply:
done in #369